auth/api/iam/openid4vp.go

```diff
@@ -158,6 +163,133 @@ func (r Wrapper) handleAuthorizeRequestFromHolder(ctx context.Context, verifier
 	}, nil
 }
 
+func (r Wrapper) handleAuthorizeRequestFromVerifier(ctx context.Context, walletDID did.DID, params map[string]string) (HandleAuthorizeRequestResponseObject, error) {
```

Comment:
shouldn't we support `request_uri`? It's `RECOMMENDED` in OpenID4VP (I think Sphereon always uses request_uri). And signed requests?

Reply:
I'll put it in a new issue. I'd rather get a first flow working, then add additional support.